docs/Java-Version-Support.adoc

```diff
@@ -1,5 +1,5 @@
 = Java Version Support
-Peter Lawrey
+Peter Lawrey, Jerry Shea
```

Comment:
suggest we remove authorship hooks altogether in general (as above for Anatomy.adoc)

Reply:
Yep - fair enough